pyspecdata/core.py

```diff
@@ -4174,10 +4184,6 @@ def argmin(self,*axes,**kwargs):
                     axis=thisindex)]
             self._pop_axis_info(thisindex)
         return self
-    def max(self):
-        return self.data.max()
-    def min(self):
-        return self.data.min()
```

Comment:
this seems to have been done in 5e5fb95a875672d.  It seems like 5e5fb95a875672d is rolling back changes that were going in the right direction.  I would revert 5e5fb95a875672d.

Reply:
so actually, in db88b74, I'm trying to fix some of those changes, so first revert db88b74 then revert 5e5fb95

Reply:
Does this need to be done in master? And by revert you mean to take the entirety of core.py and make it go back to its state in db88b74, then make it go back to its state in 5e5fb95?

Reply:
I mean than on the alec branch, I would do:
`git revert db88b74` then `git revert 5e5fb95`
then do any debugging needed to get it to work properly.

I would just do this, but I'm worried that if we both did it at the same time, etc, etc.

Reply:
to clarify, this rolls back the changes in 5e5fb95 (which itself appears to be some type of revert), since I'm claiming those are bad changes.  You need to do db88 first because I was manually correcting some of those changes, and when I tried this myself, not rolling back db88 first makes git think there is a conflict

Reply:
I see - that makes sense. I will begin this now.